protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -30,70 +32,51 @@ func BuildEtcdManifest(c *EtcdCluster) *v1.Pod {
 	pod.Kind = "Pod"
 	pod.Name = c.PodName
 	pod.Namespace = "kube-system"
+	pod.Labels = map[string]string{"k8s-app": c.PodName}
 
-	pod.Labels = map[string]string{
-		"k8s-app": c.PodName,
-	}
-
-	// TODO another hardcoded version
-	image := "/etcd:2.2.1"
-	imageRegistry := "gcr.io/google_containers"
+	etcdImage := "/etcd:2.2.1"
+	etcdRegistry := "gcr.io/google_containers"
 
-	// Test to determine if the container registry has been passed in as a flag.
-	// If so use the provider registry location.
+	// @check if the container is being overloaded via flags
 	if c.ImageSource == "" {
-		image = imageRegistry + image
+		etcdImage = etcdRegistry + etcdImage
 	} else {
-		image = strings.TrimSuffix(c.ImageSource, "/") + image
+		etcdImage = strings.TrimSuffix(c.ImageSource, "/") + etcdImage
 	}
 
 	pod.Spec.HostNetwork = true
-
 	{
 		container := v1.Container{
 			Name:  "etcd-container",
-			Image: image,
+			Image: etcdImage,
 			Resources: v1.ResourceRequirements{
 				Requests: v1.ResourceList{
 					v1.ResourceCPU: c.CPURequest,
 				},
 			},
-			Command: []string{
-				"/bin/sh",
-				"-c",
-				"/usr/local/bin/etcd 1>>/var/log/etcd.log 2>&1",
-			},
-
-			// Note that we listen on 0.0.0.0, not 127.0.0.1, so we can support etcd clusters
-			Env: []v1.EnvVar{
-				{Name: "ETCD_NAME", Value: c.Me.Name},
-				{Name: "ETCD_DATA_DIR", Value: "/var/etcd/" + c.DataDirName},
-				{Name: "ETCD_LISTEN_PEER_URLS", Value: fmt.Sprintf("http://0.0.0.0:%d", c.PeerPort)},
-				{Name: "ETCD_LISTEN_CLIENT_URLS", Value: fmt.Sprintf("http://0.0.0.0:%d", c.ClientPort)},
-				{Name: "ETCD_ADVERTISE_CLIENT_URLS", Value: fmt.Sprintf("http://%s:%d", c.Me.InternalName, c.ClientPort)},
-				{Name: "ETCD_INITIAL_ADVERTISE_PEER_URLS", Value: fmt.Sprintf("http://%s:%d", c.Me.InternalName, c.PeerPort)},
-				{Name: "ETCD_INITIAL_CLUSTER_STATE", Value: "new"},
-				{Name: "ETCD_INITIAL_CLUSTER_TOKEN", Value: c.ClusterToken},
-			},
+			Command: []string{"/usr/local/bin/etcd"},
```

Comment:
I'm happy to remove from the PR or look at at 'tee' option. It does kinda feel more naturally to use whatever docker logger method is currently selected rather than writing log files to boxes. Also i wasn't sure about who was doing the log rotation on these files?

```shell
$ docker run -ti --entrypoint=sh gcr.io/google_containers/etcd:2.2.1
/ # which tee
/bin/tee
```